blocks/blogheader/blogheader.css

```diff
@@ -0,0 +1,150 @@
+main.has-sidebar > div.blogheader-container {
+  grid-column: 2 / span 2;
+  color: var(--background-color);
+  padding: 0;
+}
+
+nav#blogheader {
+  --number-of-menu-items: 5;
+
+  padding: 0;
+  margin: 0;
+}
+
+nav#blogheader > div {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+nav#blogheader ul {
+  display: table;
+  table-layout: fixed;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  width: 100%;
+  background-color: var(--text-color);
+}
+
+nav#blogheader li a {
+  font-size: 1.4rem;
+  text-decoration: none;
+  transition: none;
+  background: inherit;
+  color: inherit;
+  font-weight: 400;
+}
```

Comment:
I think the font properties need a little bit more attention here to make it match the original.
I think it may need (at least):
```css
    font-family: var(--ff-gilroy-semibold);
    line-height: 1.714;
    font-size: 1.6rem;
    -webkit-font-smoothing: antialiased;
```

Reply:
👍